src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -63,10 +64,13 @@ private static ParticipantType GetParticipantType(bool required)
         private static bool IsValidEmail(string email)
             => new EmailAddressAttribute().IsValid(email);
 
-        public static string GenerateMeetingTitle(Invitation invitation) 
-            => $"Invitation for punch-out, IPO-{invitation.Id}, Project: {invitation.ProjectName}";
+        public static string GenerateMeetingTitle(Invitation invitation, IProjectRepository projectRepository)
```

Comment:
make this method async. Rename to GenerateMeetingTitle. The calling method CreateInvitationCommandHandler.CreateOutlookMeeting is already async, so that should not be a problem to refactor. Rename that method to CreateOutlookMeetingAsync